### ISSUE TITLE
Admin ghosts can be picked by the resurrect rune

### ISSUE THREAD

**Issue by Probe1** [d3athrow/vgstation13#17756]
#### Description of issue

I was admin-observing a cult they used a resurrection rune which put me into a players body.  I had been messaging the cult and chaplain all round through Narsie/prayers and it was highly inappropriate for me to be picked.

#### Difference between expected and actual behavior

Resurrect to check and see if someones an active admin
It did not and I was suddenly thrust into the round

#### Steps to reproduce 
Log in as an admin ghost and have someone use a resurrection rune over you

#### Specific information for locating

why do we even have so many sections 

@DeityLink

**Comment by SonixApache** [comment]
If deity doesn't fix it by tomorrow I will.
Should just be two lines to add at runes.dm#L515

**Comment by Probe1** [comment]
Please do.  In 3 years I've never been so panic stricken.

**Comment by Aurx** [comment]
Have you considered not standing on resurrect runes if you don't want to be resurrected?  Feature, not a bug.  

Just loiter on a wall tile, or use the observe verb.

**Comment by Probe1** [comment]
You're like the foil to make sure every side of a point is considered.

I was haunting a player and they drew a rune while I wasn't paying attention and had my view screen scrolled waaaaay out.  I didn't know what he was doing beforehand and it's challenging to watch a single player every second of the round let alone all of them.

The real nail in the coffin here is if you're adminning you shouldn't become an antag. Full stop.  The code allows that and shouldn't.

**Comment by Aurx** [comment]
Antagging while 'minned is a people issue, not a code issue.  We have the de-admin self verb, it works perfectly.  

Moreover, 'mins may want to roll for antag like everyone else.  There's no issue with that so long as they turn off 'min if they get lucky.  

That said, a "do not put in game" option would be a decent addition for people who want to observe without worry about runes and whatnot.

**Comment by Riekland** [comment]
making people unable to become antag while adminning is a bad idea because as it stands it's a great way to kick retards out of admin positions

**Comment by DeityLink** [comment]
@SonixApache go ahead. I don't have much time these days, so when I code I try to do progress on datums/cult3.0

On that note, cult 3.0 would fix it since it'll only allow dead cultists to come back to life (non-cultists will have another way to come back to life, both to help the cult, or to possibly save the crew, following an idea I had recently)

For laughs, how did it went @Probe1 ?

**Comment by Probe1** [comment]
@DeityLink It wasn't terribly interesting.  The players body I was forced into had two stun papers and a starter talisman which I converted into an EMP.  With about 3 minutes left until the shuttle arrived I stunned the chaplain in the halls with a fireaxe-wielding mime as backup and an IAA who I doubled up on with an EMP and stun talisman to make sure he didn't pull out a sneaky egun.  We only had 7, having left two braindeads on the asteroid, and summoning was out of the question.  The shuttle was only populated by cultists so that's something.

I felt like doodoo after spending the round talking to cultists through narsie, and the chaplain through prayer, only to become one and kill the other.
